misk/web/tabs/web-actions/src/rewrite/LinkToProtoDoc.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { useContext } from "react"
+import {ProtobufDocUrlPrefix} from "./TabContainer"
+
+interface Props {
+  protoClass: String
+}
+
+export default function LinkToProtoDoc({ protoClass }: Props) {
+  const protobufDocUrlPrefix = useContext(ProtobufDocUrlPrefix)
+
+  return (
+    <>
+      {protobufDocUrlPrefix ? (
+        <a href={protobufDocUrlPrefix + protoClass}>{protoClass}</a>
+      ) : null}
```

Comment:
does this show up with `null` written on the screen? `<span/>` might be a safer null case for React

Reply:
Good call to pull this into it's own component 👍

Reply:
Returning null tells react not the render anything: https://reactjs.org/docs/conditional-rendering.html#preventing-component-from-rendering